Description: Modify the build system in order to accommodate the removal
	of the OCR-A sources.
Author: Fabian Greffrath <fabian@debian.org>
Last-Update: 2021-08-20

--- a/configure.ac
+++ b/configure.ac
@@ -190,7 +190,7 @@ AC_PRESERVE_HELP_ORDER
 AC_CONFIG_AUX_DIR([.])
 AM_INIT_AUTOMAKE([foreign color-tests dist-zip no-dist-gzip
   parallel-tests silent-rules])
-AC_CONFIG_SRCDIR([msk-ocra.mp])
+AC_CONFIG_SRCDIR([msk-ocrbdef.mp])
 AC_CONFIG_MACRO_DIR([m4])
 AC_REVISION([$Id: configure.ac 8723 2021-03-28 18:28:09Z mskala $])
 AC_COPYRIGHT([public domain])
